README.md

```diff
@@ -1,17 +1,69 @@
-1. Project info
-This project is about creating simple Tic Tac Toe game.
+
+TicTacToeGit
+
+Simple game of Tic Tac Toe 
+
+Getting Started
+
+/*These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.*/
```

Comment:
Sorry, didn't spot it earlier.
This is markdown, not a js file. I assume using the comment notation here is a mistake.
Please [see this](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for more details on the syntax